lib/raidenclient/RaidenClient.ts

```diff
@@ -70,6 +70,22 @@ class RaidenClient extends SwapClient {
     await this.verifyConnection();
   }
 
+  public maximumOutboundCapacity = (currency: string): number => {
+    return this.maximumOutboundAmounts.get(currency) || 0;
+  }
+
+  public updateCapacity = async () => {
+    try {
+      for (const [currency] of this.tokenAddresses) {
+        const maximumOutboundAmount = (await this.channelBalance(currency)).balance;
```

Comment:
It would be nice to `await` these in parallel with `Promise.all`, particularly since this is something that will be called often.

Reply:
Think it won't matter in the context of Raiden because all the API calls are blocking. Nevertheless, I've added the recommended changes so that we're covered on xud side.

Reply:
Oh good point, but ok yeah since you've made the change it certainly doesn't hurt and one would hope these become non-blocking at some point in the future.